<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Nomaren</class>
 <widget class="QMainWindow" name="Nomaren">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>nomaren</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenMediaFile"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionMediaPlaybackPause"/>
   <addaction name="actionMediaPlaybackStart"/>
   <addaction name="actionMediaPlaybackStop"/>
   <addaction name="separator"/>
   <addaction name="actionToggleVolume"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="separator"/>
   <addaction name="actionDocumentProperties"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QToolBar" name="playbackToolBar">
   <property name="windowTitle">
    <string>playbackToolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ZoomIn"/>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionOpenMediaFile">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>Open Media File</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ZoomOut"/>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionMediaPlaybackPause">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaybackPause"/>
   </property>
   <property name="text">
    <string>Media Playback Pause</string>
   </property>
   <property name="toolTip">
    <string>Media Playback Pause</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ShortcutContext::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionMediaPlaybackStart">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
   </property>
   <property name="text">
    <string>Media Playback Start</string>
   </property>
   <property name="toolTip">
    <string>Media Playback Start</string>
   </property>
  </action>
  <action name="actionMediaPlaybackStop">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaybackStop"/>
   </property>
   <property name="text">
    <string>Media Playback Stop</string>
   </property>
   <property name="toolTip">
    <string>Media Playback Stop</string>
   </property>
  </action>
  <action name="actionDocumentProperties">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentProperties"/>
   </property>
   <property name="text">
    <string>Document Properties</string>
   </property>
   <property name="toolTip">
    <string>Document Properties</string>
   </property>
  </action>
  <action name="actionToggleVolume">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::AudioVolumeHigh"/>
   </property>
   <property name="text">
    <string>Toggle Audio Volume</string>
   </property>
   <property name="toolTip">
    <string>Toggle Audio Volume</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
